server/src/main/java/org/opensearch/action/search/GetAllPitNodeRequest.java

```diff
@@ -0,0 +1,35 @@
+
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.search;
+
+import org.opensearch.action.support.nodes.BaseNodeRequest;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+
+import java.io.IOException;
+
+/**
+ * Inner node get all pits request
+ */
+public class GetAllPitNodeRequest extends BaseNodeRequest {
+
+    public GetAllPitNodeRequest() {
+        super();
+    }
+
+    public GetAllPitNodeRequest(StreamInput in) throws IOException {
+        super(in);
+    }
+
+    @Override
+    public void writeTo(StreamOutput out) throws IOException {
+        super.writeTo(out);
+    }
+}
```

Comment:
For my understanding no custom handling needed for the new API?

Reply:
Yes since we're passing no parameter as part of get all API, currently no custom code / fields required.